### DISCUSSION TITLE
Why are the modal buttons HTML-labels at the examples?

### DISCUSSION THREAD

**Discussion by wuarmin** [saadeghi/daisyui#1775] [Q&A] [answered]
Hey,

I don't understand why the modal buttons are `<label>`'s at the examples:

What's the benefit of using label-tags...
```jsx
<div className="modal">
  <div className="modal-box w-11/12 max-w-5xl">
    <h3 className="font-bold text-lg">Congratulations random Internet user!</h3>
    <p className="py-4">You've been selected for a chance to get one year of subscription to use Wikipedia for free!</p>
    <div className="modal-action">
      <label htmlFor="my-modal-5" className="btn">Yay!</label>
    </div>
  </div>
</div>
```
...instead of button-tags?
```jsx
<div className="modal">
  <div className="modal-box w-11/12 max-w-5xl">
    <h3 className="font-bold text-lg">Congratulations random Internet user!</h3>
    <p className="py-4">You've been selected for a chance to get one year of subscription to use Wikipedia for free!</p>
    <div className="modal-action">
      <button className="btn">Yay!</button>
    </div>
  </div>
</div>
```
Thanks

**Comment by saadeghi** [comment, marked as answer]
You can use buttons but you also need to write some JS to open/close the modal.  
Examples on the site are CSS only. The label checks/unchecks the checkbox and the value of checkbox defines if modal should be open or close.  
There is also another method using anchor links (no need for label and checkbox):  
https://daisyui.com/components/modal/#modal-using-anchor-link

Let me know if you have any questions.

**Reply by wuarmin** [reply]
Thanks. Great answer!
